src/project/projecttypes.h

```diff
@@ -83,9 +83,7 @@ enum class CloudProjectVisibility {
 };
 
 struct CloudProjectInfo {
-    QUrl sourceUrl;
```

Comment:
I'll quickly tell what was my idea with this:
I thought we would have to save this information inside the mscz file (for example, in a json file like we do for audio settings etc), because we will need the source URL when opening the cloud project (to download the latest version of the project before opening, since it might have been modified and uploaded from another computer). 
Also, if we would use my idea of using the score id as the filename in the `cloud_projects` folder, we will need to read the display name from the project file, to show it correctly in "Recent Scores".
For these reasons, I thought it would be good to store those things in a separate json file in the mscz so that we can quickly retrieve them without having to read the whole score.

Reply:
I have reverted this change for now, but I doubt that storing this information in a separate file would be useful. Especially considering the fact that we still have to download this information from musescore.com...